app/javascript/modules/listingDetails/ListingDetailsPreferences.tsx

```diff
@@ -36,7 +36,9 @@ export const ListingDetailsPreferences = ({ listingID }: ListingDetailsPreferenc
 
   useEffect(() => {
     void getPreferences(listingID).then((preferences) => {
-      setPreferences(preferences)
+      setPreferences(
+        preferences?.filter((preference) => !preference.preferenceName.includes("Veterans"))
```

Comment:
We're using `includes` because there will be multiple preferences related to veterans, correct? Are we sure that the  "V" will be capitalized and that it will always be plural?

Reply:
Yeah that is correct. All of the names do contain specifically "Veterans", but I can add a lowercase to be more resilient.